app/src/pages/Play/index.tsx

```diff
@@ -45,69 +41,30 @@ function Play() {
     (player) => player.id === activeTurn?.player_id
   )
 
-  const [activeTurnPlayState, setActiveTurnPlayState] = React.useState(
-    ActiveTurnPlayState.Waiting
-  )
-
-  const startingSeconds =
-    activeTurn?.seconds_per_turn_override || currentGame.seconds_per_turn || 0
-
-  const [secondsLeft, setSecondsLeft] = React.useState(
-    activeTurn?.started_at
-      ? calculateSecondsLeft(
-          dateFromTimestamptzNow(activeTurn.started_at),
-          startingSeconds
-        )
-      : startingSeconds
-  )
-
-  // change in settings
-  React.useEffect(() => {
-    if (currentGame.seconds_per_turn) {
-      setSecondsLeft(currentGame.seconds_per_turn)
+  const getActiveTurnPlayState = () => {
+    if (activeTurn?.review_started_at) {
+      return ActiveTurnPlayState.Reviewing
     }
-  }, [currentGame.seconds_per_turn])
 
-  // for non active players
-  React.useEffect(() => {
-    if (activeTurn?.started_at !== null) {
-      setActiveTurnPlayState(ActiveTurnPlayState.Playing)
-      setSecondsLeft(
-        calculateSecondsLeft(
-          dateFromTimestamptzNow(activeTurn?.started_at),
-          startingSeconds
-        )
-      )
+    if (activeTurn?.started_at) {
+      return ActiveTurnPlayState.Playing
     }
-  }, [activeTurn?.started_at])
 
-  // new turn, reset state
+    return ActiveTurnPlayState.Waiting
+  }
+  const [activeTurnPlayState, setActiveTurnPlayState] = React.useState(
+    getActiveTurnPlayState()
+  )
   React.useEffect(() => {
-    setActiveTurnPlayState(ActiveTurnPlayState.Waiting)
-    setSecondsLeft(
-      activeTurn?.started_at
-        ? calculateSecondsLeft(
-            dateFromTimestamptzNow(activeTurn.started_at),
-            startingSeconds
-          )
-        : startingSeconds
-    )
-  }, [activeTurn?.id])
+    setActiveTurnPlayState(getActiveTurnPlayState())
+  }, [activeTurn]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
The `activeTurnPlayState` changes in here were slightly unrelated but at least fixed a weird initialization issue in which users were always thrown in the `Waiting` state. The additional `review_started_at` field enables us to compute the correct state solely from `activeTurn` which simplified some of this.

We could probably remove the other `setActiveTurnPlayState` calls below in favor of this `[activeTurn]` effect, but I just left them there for now. I don't think they're hurting anything and might make things a little more responsive (i.e. as opposed to waiting for the 1 second subscription).